LIS3DH.py

```diff
@@ -137,7 +143,7 @@ def getAxis(self, axis):
       if range==self.RANGE_2G:    divisor = 16380
       elif range==self.RANGE_4G:  divisor = 8190
       elif range==self.RANGE_8G:  divisor = 4096
-      elif range==self.RANGE_16G: divisor = 2048
+      elif range==self.RANGE_16G: divisor = 1365.33
```

Comment:
I don't actually have one of my sensors available to test with, but I agree with what you've said given the information on the datasheet, so I'll take your word for it!! If you can sort the tabs then I'd love to merge this in - thanks very much!